### PR TITLE
Add generator for logs, events and traces sources

### PR DESCRIPTION
**Pull request by sumo-drosiek**
###### Description

Auto-generate logs, events and traces endpoints in terraform
It doesn't add automatic usage of these sources in fluentd. It has to be done manually

This PR is made almost on the top of the #705 

###### Testing performed

- [ ] ci/build.sh
- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

File: deploy/kubernetes/fluentd-sumologic.yaml.tmpl [hunk 1]
```diff
@@ -993,12 +993,17 @@ spec:
...
           mountPath: /mnt/pos/
-        env:
-        - name: SUMO_ENDPOINT_LOGS_SOURCE
+        - name: SUMO_ENDPOINT_DEFAULT_LOGS_SOURCE
           valueFrom:
             secretKeyRef:
               name: sumologic
               key: endpoint-logs
+
+        - name: SUMO_ENDPOINT_DEFAULT_TRACES_SOURCE
```
Comment (sumo-drosiek) [inline deploy/kubernetes/fluentd-sumologic.yaml.tmpl:1002]: It shouldn't be here, traces are not enable 😮

**Comment by sumo-drosiek** [comment]
Forgot about upgrading docker image

File: deploy/helm/sumologic/templates/otelcol-deployment.yaml [hunk 2]
```diff
@@ -39,6 +39,8 @@ spec:
         env:
         - name: GOGC
           value: "80"
+{{- $ctx := .Values -}}
+{{ include "fluentd.envs" (dict "Context" $ctx "Type" "traces")}}
```
Comment (perk-sumo) [inline deploy/helm/sumologic/templates/otelcol-deployment.yaml:43]: Why do we call it `flutend.envs` if we use it for otelcol as well?

Comment (sumo-drosiek) [reply deploy/helm/sumologic/templates/otelcol-deployment.yaml:43]: because we didn't use it for otelcol before 😅

**Comment by sumo-drosiek** [comment]
Close and reopen due to ci

**Pull request merged**